main.py

```diff
@@ -0,0 +1,89 @@
+import os
+import sys
+
+import bandwidth
+from bandwidth import ApiException
+import click
+from InquirerPy import inquirer
+
+
+try:
+    BW_USERNAME = os.environ['BW_USERNAME']
+    BW_PASSWORD = os.environ['BW_PASSWORD']
+    BW_ACCOUNT_ID = os.environ['BW_ACCOUNT_ID']
+    BW_NUMBER = os.environ['BW_NUMBER']
+    BW_VOICE_APPLICATION_ID = os.environ['BW_VOICE_APPLICATION_ID']
+    BW_MESSAGING_APPLICATION_ID = os.environ['BW_MESSAGING_APPLICATION_ID']
+except KeyError:
+    print("Please set the environmental variables defined in the README")
+    sys.exit(1)
+
+bandwidth_configuration = bandwidth.Configuration(
+    username=BW_USERNAME,
+    password=BW_PASSWORD
+)
+
+bandwidth_api_client = bandwidth.ApiClient(bandwidth_configuration)
+bandwidth_mfa_api_instance = bandwidth.MFAApi(bandwidth_api_client)
+
+
+@click.command()
+def send_and_verify_mfa_code():
+    code_expiry_time = 3
+
+    phone_number = inquirer.text(
+        message="Please enter the phone number to receive the MFA code, in E164 format (ex: +15555555555)",
+        validate=lambda x: x.startswith("+1") and len(x) == 12,
```

Comment:
do we need to make sure this is 12 characters long? I think you can have a valid E164 number that's not that long

Reply:
i guess not, i was actually just thinking US but this will work with the new GBR numbers, so ill remove that